sopel/modules/instagram.py

```diff
@@ -139,3 +148,12 @@ def parse_insta_json(json):
 
     # Build the message
     return ' | '.join(parts)
+
+
+def parse_oembed_json(json):
+    if not json:
+        raise ParseError("No valid JSON returned")
+    return "[insta] Post by {} | {}".format(
```

Comment:
How about noting that this is fallback/limited parsing somehow?

```suggestion
    return "[insta|fallback] Post by {} | {}".format(
```

Reply:
As discussed - at length - on IRC, we couldn't really settle on a format here that makes complete sense without confusing _someone_, and the existing parsing method logic already supports a `[insta] Post by user | Title` format if the other parts aren't found in the data. Thus I think sticking with my original output here is for the "best".

Perhaps this can be re-visited later.